CONTRIBUTING.md

```diff
@@ -0,0 +1,150 @@
+
+2. Fork the repository associated with the issue to your local GitHub account. This means that you will have a copy of the repository under your-GitHub-username/repository-name.
+
+3. Clone the repository to your local machine.
+
+    ```bash
+    git clone https://github.com/OSCA-Kampala-Chapter/autobot
+    ```
+
+4. Create a new branch for your fix.
+
+    ```bash
+    git checkout -b branch-(name-here)
+    ```
+
+5. Make the appropriate changes for the issue you are trying to address or the feature that you want to add.
+
+6. Add the file contents of the changed files to the "snapshot" git uses to manage the state of the project, also known as the index.
+
+    ```bash
+    git add <file_list>
+    ```
+
+7. Insert a short message of the changes made to store the contents of the index with a descriptive message.
+
+    ```bash
+    git commit -m "Insert a short message of the changes made here"
+    ```
+
+8. Push the changes to the remote repository
+
+    ```bash
+    git push origin branch-name-here.
+    ```
+
+9. Submit a pull request to the upstream repository.
+
+10. Title the pull request with a short description of the changes made and the issue or bug number associated with your change. For example, you can title an issue like so "Added more log outputting to resolve #105".
+
+11. In the description of the pull request, explain the changes that you made, any issues you think exist with the pull request you made, and any questions you have for the maintainer. It's OK if your pull request is not perfect (no pull request is), the reviewer will be able to help you fix any problems and improve it!
+
+12. Wait for the pull request to be reviewed by a maintainer.
+
+13. Make changes to the pull request if the reviewing maintainer recommends them.
+
+14. Celebrate your success after your pull request is merged!
+
+15. Most importantly,
+  - __Be consistent in the use of these guidelines when submitting.__
+  - __now!__
+  - Happy coding!
+
+#### Other Requirements for Submissions
+- The file extension for code files should be `.py`. 
+- Strictly use snake_case (underscore_separated) in your file_name, as it will be easy to parse in future using scripts.
+- Please avoid creating new directories if at all possible. Try to fit your work into the existing directory structure.
+- If possible, follow the standard *within* the folder you are submitting to.
+- If you have modified/added code work, make sure the code compiles before submitting.
+- If you have modified/added documentation work, ensure your language is concise and contains no grammar errors.
+- Do not update the README.md or DIRECTORY.md file which will be periodically autogenerated by our GitHub Actions processes/MAintainers.
+- Add a corresponding explanation to [Algorithms-Explanation](https://github.com/TheAlgorithms/Algorithms-Explanation) (Optional but recommended)
```

Comment:
i think stuff related to the Alogorithms repo is irrelevant

Reply:
Resolved [solution here](https://github.com/OSCA-Kampala-Chapter/autobot/pull/114/commits/436c4620f41f3964f69aa28f49e083871b687d72)